lib/plugins/aws/package/compile/events/iotFleetProvisioning/index.test.js

```diff
@@ -0,0 +1,170 @@
+'use strict';
+
+const expect = require('chai').expect;
+const AwsProvider = require('../../../../provider/awsProvider');
+const AwsCompileIotFleetProvisioningEvents = require('./index');
+const Serverless = require('../../../../../../Serverless');
+
+describe('AwsCompileIoTEvents', () => {
+  let serverless;
+  let awsCompileIoTFleetProvisioningEvents;
+
+  beforeEach(() => {
+    serverless = new Serverless();
```

Comment:
Let's construct new tests strictly with `runServerless`. See: https://github.com/serverless/serverless/tree/master/test#unit-tests

We strand away from tests written as here, as they're very problematic for refactors (they depend on internals, when they should not)